package.json

```diff
@@ -52,6 +52,7 @@
     "rollup": "^2.56.0",
     "semver": "^7.3.5",
     "ts-node": "^9.0.0",
-    "typescript": "^4.1.0"
+    "typescript": "^4.1.0",
+    "graphql": "^16.2.0"
```

Comment:
Hoisting this here so that it's all packages are on a common `graphql` dependency. This is important during testing so that there aren't conflicting graphql ts types between packages because then `tsc` will say error with incompatible types.